thumbor/server.py

```diff
@@ -53,18 +53,6 @@ def get_config(config_path, use_environment=False):
 
     return Config.load(config_path, conf_name='thumbor.conf', lookup_paths=lookup_paths)
 
-
-def configure_log(config, log_level):
-    if (config.THUMBOR_LOG_CONFIG and config.THUMBOR_LOG_CONFIG != ''):
```

Comment:
THUMBOR_LOG_CONFIG consists only of THUMBOR_LOG_FORMAT and THUMBOR_LOG_DATE_FORMAT which are not relevant when formatting JSON